File: viasoft.qualidade.rnc.core/Viasoft.Qualidade.RNC.Core.Host/Proxies/Producao/OrdensProducao/Dtos/OrdemProducaoOutput.cs
using System;

namespace Viasoft.Qualidade.RNC.Core.Host.Proxies.Producao.OrdensProducao.Dtos;

public class OrdemProducaoOutput
{
    public int NumeroOdf { get; set; }

    public string Revisao { get; set; }
    public Guid IdProduto { get; set; }
    public decimal Quantidade { get; set; }
    public DateTime? DataInicio { get; set; }
    public DateTime DataEntrega { get; set; }
    public string NumeroPedido { get; set; }
    public string Observacao { get; set; }
    public bool IsRetrabalho { get; set; }
    public int? NumeroOdfDestino { get; set; }
    public bool OdfFinalizada { get; set; }
    public bool PossuiPartida { get; set; }
}

File: QA/viasoft.qualidade.rnc-frontend/libs/rnc-lib/src/lib/api-clients/Ordem-Producao/model/ordem-producao-output.ts
export interface OrdemProducaoOutput {
  numeroOdf: number;
  revisao: string;
  idProduto: string;
  idPedido: string;
  quantidade: number;
  dataInicio: Date;
  dataEntrega: Date;
  numeroPedido: string;
  observacao: string;
  isRetrabalho: boolean;
  numeroOdfDestino: number | null;
  numeroOdfFaturamento: number;
  idProdutoFaturamento: string;
  numeroLote: string;
}
